### DISCUSSION TITLE
Accessing file that is an archive with password protection

### DISCUSSION THREAD

**Discussion by 8hAheWMxqz** [stashapp/stash#5561] [Q&A]
I'm using docker to host stash on windows machine A.
My files are on QNAP nas - machine B.

Some files can be easliy accesed by stash to use - some videos, manga, image sets, etc.
However, I want to have some galleries with multiple layers of protection, so they're actually archives with password.

Is it possible for stash to read this kind of file as image set galleries? Of course, I expect that to access that kind of file through stash, I somehow have to provide password for stash to use.

**Comment by stg-annon** [comment]
the solution would be to unarchive the file with the password then have stash scan it, if you're looking to secure access to your files I would look at other mechanisms that accomplish this
